docker-compose.yml

```diff
@@ -111,15 +121,63 @@ services:
     volumes:
       - ./docker/nginx.conf:/etc/nginx/nginx.conf
     ports:
-      - '8080:80'
+      - '80:80'
+  generic-worker-standalone:
+    image: taskcluster/generic-worker:local
+    networks:
+      - local
+    environment:
+      TASKCLUSTER_ROOT_URL: http://taskcluster
+      TASKCLUSTER_CLIENT_ID: static/generic-worker-compose-client
+      TASKCLUSTER_ACCESS_TOKEN: j2Z6zW2QSLehailBXlosdw9e2Ti8R_Qh2M4buAEQfsMA
+    build:
+      context: ./workers
+      dockerfile: Dockerfile
+    volumes:
+      - ./docker/generic-worker-config.json:/etc/generic-worker/config.json
+      - ./docker/worker-runner-config.json:/etc/generic-worker/worker-runner.json
+    command: standalone
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      auth-web:
+        condition: service_healthy
+      queue-web:
+        condition: service_healthy
+      taskcluster:
+        condition: service_started
+  generic-worker-static:
+    image: taskcluster/generic-worker:local
+    networks:
+      - local
+    environment:
+      TASKCLUSTER_ROOT_URL: http://taskcluster
+      TASKCLUSTER_CLIENT_ID: static/generic-worker-compose-client
+      TASKCLUSTER_ACCESS_TOKEN: j2Z6zW2QSLehailBXlosdw9e2Ti8R_Qh2M4buAEQfsMA
+    build:
+      context: ./workers
+      dockerfile: Dockerfile
+    volumes:
+      - ./docker/generic-worker-config.json:/etc/generic-worker/config.json
+      - ./docker/worker-runner-config.json:/etc/generic-worker/worker-runner.json
+    command: static
+    depends_on:
+      rabbitmq:
+        condition: service_healthy
+      auth-web:
+        condition: service_healthy
+      queue-web:
+        condition: service_healthy
+      taskcluster:
+        condition: service_started
```

Comment:
It _looks like_ almost all of these two services are the same, aside from the `command`. It _could be_ cleaned up a bit with yaml anchors. This makes it more apparent to the reader/developer that they have the same config aside from the `command`. Up to you though!

Reply:
Ah I also just realized you'd need to do that in the generator, so if it's not too straightforward, don't worry about it :)

Reply:
:) Let's keep it, I'm pretty sure we'll find some inconsistencies with those dependencies once we start develop real features with it